src/llm/openai_client.py

```diff
@@ -6,8 +6,10 @@ class OpenAI:
     def __init__(self):
         config = Config()
         api_key = config.get_openai_api_key()
+        base_url = config.get_openai_base_url()
         self.client = OAI(
             api_key=api_key,
+            base_url=base_url
         )
```

Comment:
**Suggestion**: Since the OpenAI client now accepts a base_url parameter, ensure that all calls to the OpenAI API throughout the codebase are updated to use this new configuration option. This consistency will help avoid hardcoding URLs in multiple places. <br> **Code Suggestion**: 
 ```
 -        self.client = OAI(api_key=api_key)
 +        self.client = OAI(api_key=api_key, base_url=base_url)
 ```